pkg/apis/controlplane/types.go

```diff
@@ -330,6 +331,14 @@ type HTTPProtocol struct {
 	Path string
 }
 
+// TLSProtocol matches TLS handshakes with specific SNI. If the field is not provided, this
+// matches all TLS handshakes.
+type TLSProtocol struct {
+	// SNI indicates the domain name in the TLS hello message, allowing the server to match it with the correct digital
+	// certificate for that domain.
```

Comment:
"allowing the server to match it with the correct digital certificate for that domain."
I don't think we have this functionality? We just check if the actual SNI matches the one set in the policy.

Reply:
Removed `allowing the server to match it with the correct digital certificate for that domain.`